src/main/g8/build.sbt

```diff
@@ -0,0 +1,11 @@
+import Dependencies._
+
+lazy val root = (project in file(".")).
+  settings(
+    inThisBuild(List(
+      organization := "com.example",
+      scalaVersion := "2.12.2"
+    )),
+    name := "$name$",
+    libraryDependencies ++= Seq(akkaHttp, akkaHttpTestkit % Test, scalaTest % Test)
```

Comment:
should we use a less programatic looking build, i.e. just separate lines in build.sbt?